src/main/java/frc/robot/Subsystems/ElevatorSubsystem.java

```diff
@@ -85,9 +107,11 @@ public void periodic() {
       SmartDashboard.putNumber("elevator/motor 16/output power", elevatorMotor2.getAppliedOutput());
       SmartDashboard.putNumber("elevator/motor 17/raw output power", elevatorMotor1.get());
       SmartDashboard.putNumber("elevator/motor 16/raw output power", elevatorMotor2.get());
+
+      moveElevator(this.elevatorSetpoint);
```

Comment:
Can you help me understand why we want repeated (periodic) calls to this? We're not adjusting this.elevatorSetpoint in this method, so why do we need the periodic calls?